import_tables.sql

```diff
@@ -22,15 +22,16 @@ SELECT
 FROM source_pluto
 INNER JOIN borough ON source_pluto.borough=borough.abbr;
 
-COPY zoning_district (
-	"id",
-	"label",
-	"wgs84",
-	"li_ft"
-)
-	FROM '../zoning_district.csv'
-	DELIMITER ','
-	CSV HEADER;
+INSERT INTO zoning_district
+SELECT
+    GEN_RANDOM_UUID() AS id,
+    zonedist AS label,
+    ST_Transform(wkt, 4326) as wgs84,
+		ST_Transform(wkt, 2263) as li_ft
+FROM source_zoning_districts
+WHERE
+    zonedist NOT IN ('PARK', 'BALL FIELD', 'PUBLIC PLACE', 'PLAYGROUND', 'BPC', '');
+    -- AND ST_GEOMETRYTYPE(ST_MAKEVALID(wkt)) = 'ST_MultiPolygon';  This doesn't seem to do anything, but was in data-engineering/products/pluto/pluto_build/sql/export_ae_tables.sql
```

Comment:
Huh... looks like `ST_MAKEVALID` checks to see that `wkt` is valid and if not, coerces it to be a valid geom. The `ST_GEOMETRYTYPE` part seems to make it so that we only want mulitpolygons. I'm wondering if the inclusion of these checks means that we have to be extra careful about source zoning districts data...?

Reply:
So, we already know it's a valid geometry type, so we can skip that, and just make sure it's a multipolygon.  I've made that update.